doc/source/security.md

```diff
@@ -0,0 +1,80 @@
+# Security
+
+The information in this document focuses primarily on cloud based deployments. For on-premise deployments, additional security work that is specific to your installation method would also be required. Note that your specific installation's security needs might be more or less stringent than what we can offer you here.
+
+Brad Geesamen gave a wonderful talk titled [Hacking and Hardening Kubernetes by Example](https://kccncna17.sched.com/event/CU6z/hacking-and-hardening-kubernetes-clusters-by-example-i-brad-geesaman-symantec) at Kubecon NA 2017. You can [watch the talk](https://www.youtube.com/watch?v=vTgQLzeBfRU) or [read the slides](https://schd.ws/hosted_files/kccncna17/47/Hacking%20and%20Hardening%20Kubernetes%20By%20Example%20v1.pdf). Highly recommended that you do so to understand the security issues you are up against when using Kubernetes to run JupyterHub.
+
+## Secure the access to Helm
+
+In its default configuration, helm pretty much allows root access to all other
+pods running in your cluster. See this [Bitnami Helm security article](https://engineering.bitnami.com/articles/helm-security.html)
+for more information. As a consequence, the default allows all users in your cluster to pretty much have root access to your whole cluster! 
+
+You can mitigate this by limiting public access to the Tiller API. To do so, use the following command:
+
+```bash
+kubectl --namespace=kube-system patch deployment tiller-deploy --type=json --patch='[{"op": "add", "path": "/spec/template/spec/containers/0/command", "value": ["/tiller", "--listen=localhost:44134"]}]'
```

Comment:
wonder if it's worth highlighting that this namespace is not whatever namespace that they chose when setting up their jupyterhub